cumulusci/tasks/salesforce/activate_flow.py

```diff
@@ -0,0 +1,59 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.utils import process_list_arg
+from cumulusci.core.exceptions import TaskOptionsError
+
+
+class ActivateFlow(BaseSalesforceApiTask):
+    """
+    Activate the Flows with the supplied Developer Names
+    """
+
+    task_options = {
+        "developer_names": {
+            "description": "List of DeveloperNames to query in SOQL",
+            "required": True,
+        },
+    }
+
+    def _init_options(self, kwargs):
+        super(ActivateFlow, self)._init_options(kwargs)
+        self.options["developer_names"] = process_list_arg(
+            self.options.get("developer_names")
+        )
+        self.api_version = "43.0"
+        if not self.options["developer_names"]:
+            raise TaskOptionsError(
+                "Error you are missing developer_names definition in your task cumulusci.yml file. Please pass in developer_names for your task configuration or use -o to developer_names as a commandline argument"
+            )
```

Comment:
This line is missing test coverage. It should be pretty easy to add: just a test that tries to create the task with developer_names set to an empty list, and asserts that that raises a TaskOptionsError.